src/UKHO.S100PermitService.API/Program.cs

```diff
@@ -113,9 +112,10 @@ private static void ConfigureServices(WebApplicationBuilder builder)
             builder.Services.AddEndpointsApiExplorer();
             builder.Services.AddDistributedMemoryCache();
 
-            builder.Services.Configure<EventHubLoggingConfiguration>(builder.Configuration.GetSection("EventHubLoggingConfiguration"));
-            builder.Services.Configure<HoldingsServiceApiConfiguration>(configuration.GetSection(HoldingsServiceApiConfiguration));
-            builder.Services.Configure<UserPermitServiceApiConfiguration>(configuration.GetSection(UserPermitServiceApiConfiguration));
+            builder.Services.Configure<EventHubLoggingConfiguration>(configuration.GetSection("EventHubLoggingConfiguration"));
+            builder.Services.Configure<HoldingsServiceApiConfiguration>(configuration.GetSection("HoldingsServiceApiConfiguration"));
+            builder.Services.Configure<UserPermitServiceApiConfiguration>(configuration.GetSection("UserPermitServiceApiConfiguration"));
+            builder.Services.Configure<ProductKeyServiceApiConfiguration>(configuration.GetSection("ProductKeyServiceApiConfiguration"));
```

Comment:
Please move all of these configuration strings into a private const string definition in Program.cs

Reply:
Resolved